piksi_multi_rtk_gps/startup/rc.local

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# rc.local
+#
+# This script is executed at the end of each multiuser runlevel.
+# Make sure that the script will "exit 0" on success or any other
+# value on error.
+#
+# In order to enable or disable this script just change the execution
+# bits.
+#
+# By default this script does nothing.
+
+#run rtk on startup
+su - tradr -c 'source ~/catkin_ws/devel/setup.bash; roslaunch ~/catkin_ws/src/mav_rtk_gps/piksi_multi_rtk_gps/startup/startup.launch &'
```

Comment:
What about placing this in the `mav_eurathlon` folder? It has some explicit dependency on the setup for the challenge, like the username and path of the catkin workspace.